### ISSUE TITLE
the client will automatically log out and needs SMS verification after around 12 hours due to session expiration.

### ISSUE THREAD

**Issue by ypxie** [jmfernandes/robin_stocks#62]
Is there anyway to work around this?

**Comment by mingwugmail** [comment]
+1, same problem.

**Comment by ImpetusSports** [comment]
I have a solution for this where I only logged in once in release 0.9.8.8.  I kept the device_token from my first log in (can remember if i made a custom log in class to get that) and can call the default robinhood.login(username, password, device_token=token, expiresIn=999999999999999).  Since I have the device token, I don't have to do the sms verification.  I have been able to use this token for the past couple months without logging in.  Not sure when the implementation of the new login method was introduced (just tried updating today and noticed this issue when running my unit tests) but, it no longer allows the user to pass through the device_token.
